Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -436,4 +436,40 @@ void DebugMsg(const char* fmt, ...);
 #define  DEFAULT_FATAL(var)  default: FATAL("No valid case for switch variable '" #var "'"); break;
 #endif
 
+#if defined(__clang__) || defined(__GNUC__)
+#   define _COMPILER_WARNING_NAME(warning) "-W" warning
+#   define _COMPILER_STRINGIZE(exp) #exp
```

Comment:
This should probably be moved outside the `__clang__ || __GNUC__` block, perhaps above with `hsMacroJoin()`?